deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml

```diff
@@ -34,11 +34,12 @@ spec:
       serviceAccountName: snapshot-controller
       containers:
         - name: snapshot-controller
-          image: registry.k8s.io/sig-storage/snapshot-controller:v8.2.0
+          # TODO: Update this to be dynamic
+          image: quay.io/niryadav/snapshot-controller:latest
```

Comment:
Please suggest the correct URL to use here..

Reply:
Let's not change it, the d/s build will take care of updating it while building the bundle.
We can leave it as it is.